README.md

```diff
@@ -462,6 +463,28 @@ to live.
 
 Note: `chronos-sync.rb` does not delete jobs by default. You can pass the `--delete-missing` flag to `chronos-sync.rb` to remove jobs. Alternatively, you can manually remove it using the API or web UI.
 
+## Mesos Framework Authentication
+
+To enable framework authentication in Chronos:
+
+* Run Chronos with `--mesos_authentication_principal` set to a Mesos-authorized principal. For Mesos' built-in CRAM-MD5 authentication, you must also provide `--mesos_authentication_secret_file` pointing to a file containing your authentication secret.
+
+> The secret file cannot have a trailing newline. To not add a newline simply run:
+```bash
+$ echo -n "secret" > /path/to/secret/file
+```
+
+* If using the built-in CRAM-MD5 authentication mechanism, run mesos-master with the credentials flag and the path to file with authorized users + their secrets.
+`--credentials=/path/to/credential/file`
```

Comment:
Did you intend this to go on a newline? Because it doesn't when rendered at https://github.com/mesos/chronos/tree/credential#mesos-framework-authentication